### ISSUE TITLE
Refactoring Extract an Interface should include xml comments

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#30948]
<p>The Extract an Interface does not include XML comments. Considering the purpose of the refactoring is to generate something to share it should include XML comments to maximise the value of the interface.<br><br>Resharper includes comments. No need for an option, this should be default behaviour. Why wouldn't you want to document your interface?</p>

_This issue has been moved from https://developercommunity.visualstudio.com/content/idea/358771/refactoring-extract-an-interface-should-include-xm.html
VSTS ticketId: 706627_
_These are the original issue comments:_

Fiona Niu[MSFT] on 10/16/2018, 07:19 PM (18 days ago): <p>Thank you for taking the time to provide your suggestion. We will do some preliminary checks to make sure we can proceed further.&nbsp;You will hear from us in about a week on our next steps. </p>
Kendra Havens [MSFT] on 10/31/2018, 03:26 PM (3 days ago): <p>Your suggestion has been queued up for prioritization.  Feature suggestions are prioritized based on the value to our broader developer community and the product roadmap. We may not be able to pursue this one immediately, but we will continue to monitor it up to 90 days for community input</p>

**Comment by koolraap** [comment]
The original issue had a link to VS doco and the word Refactoring giving a bit more context. "The [Extract an Interface Refactoring](https://docs.microsoft.com/en-au/visualstudio/ide/reference/extract-interface?view=vs-2017)..."

**Comment by scott-moxham** [comment]
This is badly needed - I waste so much time extracting an interface and then having to go through and manually copy all of the XML comments to the new interface; this is something that is much better suited to automation than wasting developer time. I would consider this to be one of the biggest wastes of time that I experience while developing.

**Comment by JadaVonRuth** [comment]
Yes I totally agree, it wastes time to transfer them manually. This should be default behavior or at least it should be an option. It should transfer xml docs and replace original ones with <inheritdoc />
